### PR TITLE
Added unit tests for BKG events controller.

### PR DESCRIPTION
**Pull request by douradofrazer**

### CODE REVIEW

File: src/test/java/org/dcsa/bkg/controller/BKGEventControllerTest.java [hunk 1]
```diff
@@ -0,0 +1,273 @@
+package org.dcsa.bkg.controller;
```
Comment (HenrikHL) [inline src/test/java/org/dcsa/bkg/controller/BKGEventControllerTest.java:1]: IS the idea with just "just" to get started? We are missing test cases for all positive results...

Comment (douradofrazer) [reply src/test/java/org/dcsa/bkg/controller/BKGEventControllerTest.java:1]: There are positive test cases as well in this.

Comment (douradofrazer) [reply src/test/java/org/dcsa/bkg/controller/BKGEventControllerTest.java:1]: Do you mean for each request param as well?

Comment (HenrikHL) [reply src/test/java/org/dcsa/bkg/controller/BKGEventControllerTest.java:1]: That would be my expectation. If for some reason the code is removed so a param is no longer valid - how will our tests catch this?

**Review by nt-as — APPROVED** [review]

**Pull request merged**